.rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+ignore = [
+       "*",
```

Comment:
This pattern is no longer correct, as the [format has changed to match that of `.gitignore`](https://github.com/rust-lang/rustfmt/blob/e0664662b49c1ed3e99391c51132ddf01cd87c79/Configurations.md#ignore).

The ignore list feature isn't stable yet anyway and is only available in nightly, so I suppose it [could change again](https://github.com/rust-lang/rustfmt/issues/3243).